charts/oes/charts/spinnaker/templates/configmap/spin-config.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "spinnaker.fullname" . }}-spin-config
+  labels:
+{{ include "spinnaker.standard-labels" . | indent 4 }}
+data:
+  config: |
+    gate:
+      endpoint: http://spin-gate:8084
+    auth:
+      enabled: true
+      basic:
```

Comment:
Why are these things hardcoded?

Reply:
These are now mapped as the config map instead of hardcoding